ui/app/helpers/editable-variable-link.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
```

Comment:
The backing logic for the `editable-variable-link.hbs` component. Could have set this up as a component script, rather than a helper, but I thought we might want to use this in a different context sometime.